src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsServiceTest.java

```diff
@@ -137,7 +137,7 @@ public void islBfdPropertiesChanged(Endpoint source, Endpoint destination) {
     }
 
     @Test
-    public void shouldUpdateLinkUnderMaintenanceFlag() throws SwitchNotFoundException {
+    public void updateLinkUnderMaintenanceFlag() throws SwitchNotFoundException {
```

Comment:
Aren't you afraid of solving bunch of conflicts between this PR and the ones with migration to Junit5? May be it's better to move this renaming to migration PR?

Reply:
The base of this PR is Junit migration patch, so This PR will be merged after Junit